Samples/Client/SwiftClient/SwiftClientTests/SwiftClientTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SwiftClientTests.swift
+//  SwiftClientTests
+//
+//  Created by Michael Perel on 9/24/17.
+//  Copyright © 2017 Michael Perel. All rights reserved.
+//
+
+import XCTest
+@testable import SwiftClient
+
+class SwiftClientTests: XCTestCase {
```

Comment:
are you planning to add these tests? @michaelperel

Reply:
Cocoa Pods automatically generates these. I plan on writing these but need a little bit of guidance as to what we should test specifically. The plan was to make sure the requests from the iOS client are the same as the requests from the web client, assuming that the web client is the canonical reference.

Reply:
i'd love to have some unit-test style coverage of the different components here, but it may require some changes to your architecture, that i'm not sure is in scope at this time (for instance, if we could DI inject a component responsible for issuing web requests, we could swap that out and mock for testing). In general, if we could include any component level tests that don't depend on external system state, that would be awesome